### ISSUE TITLE
Could we use deepsort to improve the result of detection ? 

### ISSUE THREAD

**Issue by le02146** [pmj110119/YOLOX_deepsort_tracker#20]
My task is use yolox to run object detect with YoloX, instead of tracking. 
I trained yolox on many images with bboxes of car, then run car detection on a video. 
But some of the frames are missing, may I use deepsort method to improve the result of detection? such as predict the bbox when a frame has no car.
Thanks.

**Comment by pmj110119** [comment]
Yes. Deepsort employs Kalman filtering to predict the positions of bounding boxes from previous frames in the current frame. A track target is only discarded if it has been consistently missed multiple times.

In offline processing, for targets that have been intermittently missed, you can utilize the Kalman filter's predictions to complement the detection results.

**Comment by le02146** [comment]
@pmj110119 Thank you for your help so much! 
My knowledge limits to object detection on images.
If I want to run offline processing for Yolox detection results of videos, should the Kalman filter or deepsort be `trained` on a video tracking dataset?

**Comment by pmj110119** [comment]
You don't need to train specifically for tracking tasks, you can use them directly

**Comment by le02146** [comment]
could you give me a hint, which func should I use ? Thank you so much.
